java-manta-client/src/main/java/com/joyent/manta/http/EncryptionHttpHelper.java

```diff
@@ -866,4 +989,34 @@ private Cipher buildMetadataDecryptCipher(final SupportedCipherDetails cipherDet
         }
         return metadataCipher;
     }
+
+    /**
+     * Converts a nullable {@link Long} array into a long primitive array.
```

Comment:
Is this saying that `ranges` can be `null`? Or that some of the values can be `null`?